src/renderer/ScreenController.tsx

```diff
@@ -211,7 +207,7 @@ export default class ScreenController extends Component {
   }
 
   showAbout() {
-    this.openDialog('About')
+    this.context.openDialog(About)
```

Comment:
The new API disallows strings to identify dialogues and only accepts (type-checked) React elements, this was already possible before but not consistently used across the code-base